Assets/Scripts/CandySwapper.cs

```diff
@@ -0,0 +1,160 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CandySwapper : MonoBehaviour
```

Comment:
Good refactor 👍 This class shows what the single responsibility principle means.

Reply:
Thank you!